gitlint-core/gitlint/tests/contrib/rules/test_authors_commit.py

```diff
@@ -0,0 +1,88 @@
+from collections import namedtuple
```

Comment:
Thanks for taking the time to write tests 👍 

We should however match existing test styling. While there's obviously nothing wrong with writing pytest-style tests (I do so myself in other projects), we should stay consistent within the project and not do so here. Even if we were to agree that pytest-style tests are objectively better (a debate in itself), for long-term maintainability, consistency is more important. Thanks for understanding.


This includes not using `@pytest.mark.parametrize`, and not using `assert` statements.
I recommend looking at the unit test of an existing contrib rule:

Reply:
Thanks for your feedback :+1:

My bad. I created this rule as part of another project, and I didn't think about the usage of `pytest`.
I agree with you that consistency is important.
I hope that the tests now fit the style of the other tests better.

Thanks for your great guidance!

Reply:
Thanks for your feedback :+1:

My bad. I created this rule as part of another project, and I didn't think about the usage of `pytest`.
I agree with you that consistency is important.
I hope that the tests now fit the style of the other tests better.

Thanks for your great guidance!